Mage.Sets/src/mage/cards/a/AnnexSentry.java

```diff
@@ -0,0 +1,59 @@
+package mage.cards.a;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.EntersBattlefieldTriggeredAbility;
+import mage.abilities.common.delayed.OnLeaveReturnExiledToBattlefieldAbility;
+import mage.abilities.effects.common.CreateDelayedTriggeredAbilityEffect;
+import mage.abilities.effects.common.ExileUntilSourceLeavesEffect;
+import mage.abilities.keyword.ToxicAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.ComparisonType;
+import mage.constants.SubType;
+import mage.filter.common.FilterOpponentsArtifactCreaturePermanent;
+import mage.filter.predicate.mageobject.ManaValuePredicate;
+import mage.target.TargetPermanent;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author AhmadYProjects
+ */
+public final class AnnexSentry extends CardImpl {
+
+    private static final FilterOpponentsArtifactCreaturePermanent filter = new FilterOpponentsArtifactCreaturePermanent("creature with cmc 3 or less");
```

Comment:
you don't need to create a whole new class for this, just add the necessary predicates to a filter that already exists, also it needs to be artifact or creature and not artifact creature

Reply:
so something like this?

private static final FilterPermanent filter = new FilterPermanent("Artifact or creature an opponent controls with cmc 3 or less");

    static {
        filter.add(new ManaValuePredicate(ComparisonType.FEWER_THAN,4));
        filter.add(Predicates.or
                (CardType.ARTIFACT.getPredicate(),
                CardType.CREATURE.getPredicate()));
        filter.add(TargetController.OPPONENT.getControllerPredicate());
    }

Reply:
yeah, except the filter message shouldn't be capitalized